packages/inferno-component/src/index.ts

```diff
@@ -19,19 +27,20 @@ import VNodeFlags from 'inferno-vnode-flags';
 let noOp = ERROR_MSG;
 
 if (process.env.NODE_ENV !== 'production') {
-	noOp = 'Inferno Error: Can only update a mounted or mounting component. This usually means you called setState() or forceUpdate() on an unmounted component. This is a no-op.';
+	noOp =
+		'Inferno Error: Can only update a mounted or mounting component. This usually means you called setState() or forceUpdate() on an unmounted component. This is a no-op.';
```

Comment:
GitHub displays tabs with 8 spaces. Here we just indent it because it goes over 80 columns.